packages/core/src/initialization/make-index-api.ts

```diff
@@ -40,10 +40,5 @@ export function makeIndexApi(
     return undefined
   }
   const indexApi = buildIndexing(indexingConfig, logger, network)
-  if (isNonProduction(indexApi, network, indexingConfig.models)) {
```

Comment:
why'd we lose this warning?  We should just adapt it to not depend on models being set in the config file, but the intention behind this warning is still valid.  Can we put this back in a follow-up?